Source/Diagnostics/ReducedDiags/BeamRelevant.cpp

```diff
@@ -125,7 +125,7 @@ void BeamRelevant::ComputeDiags (int step)
     auto const species_names = mypc.GetSpeciesNames();
 
     // inverse of speed of light squared
-    Real constexpr inv_c2 = 1.0 / (PhysConst::c * PhysConst::c);
+    Real constexpr inv_c2 = amrex::Real(1.0) / (PhysConst::c * PhysConst::c);
```

Comment:
You should be able to do this, if you want to be more compact:
```suggestion
    Real constexpr inv_c2 = 1.0_rt / (PhysConst::c * PhysConst::c);
```